website/docs/components/rich-tooltip/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,216 @@
+## Usage
+
+### When to use
+
+- For adding extra information to a UI element that consists of more than a simple sentence, or structured content with a specific layout.
+- When it’s necessary to _temporarily persist_ extra information in the UI; e.g., if the user benefits from cross-referencing information in the component with other elements in the UI.
+
+!!! Info
+
+**Temporarily persist:** by this we mean the component will remain visible within UI until it is explicitly dismissed by the user, allowing the toggling open of the component and cross-referencing the content it contains with the rest of the UI. More details on this can be found in the [interaction](#interaction) section.
+!!!
+
+### When not to use
+
+- For extra information that consists of basic text, consider using the [Tooltip](/components/tooltip) instead.
+- For complex functions that change or alter the UI, consider a [Dropdown](/components/dropdown) instead.
```

Comment:
Wondering if we need to draw a comparison with the dropdown here? I can't imagine (but maybe I'm wrong) that consumers will confuse when to use the dropdown and when to use the richtooltip?

Reply:
One of the potential use cases that we identified was placing small "micro-interactions" in the component, like toggling things on/off, things like that. While we don't have clear examples of consumers do this, there's a bit of potential overlap in the visual language between the two I think it's useful to call out.